### ISSUE TITLE
Infinity reload Issue

### ISSUE THREAD

**Issue by shariful-xs** [prevwong/craft.js#506]
I wish I want to make wordpress plugin using this package.when I build command and all js files enque on Wordpress all is right but problem is when browser get all script file browser infinity reloading.
Is possible to solve this problem?please tell me about this.

**Comment by ismajl-ramadani** [comment]
I don't think this is related to craft.js. Also more context is needed.  Also if its general help I would recommend to join the discord channel and discuss the question there.

@prevwong  can we close this issue please.
